main.go

```diff
@@ -255,9 +270,40 @@ func Teardown(Client *scw.Client, Region scw.Region, PathRegistry string) (*cont
 
 }
 
+func getKeyValue(key string) map[string]string {
+	KeyValue := make(map[string]string)
+	EnvironmentKeyValues := strings.Split(os.Getenv(key), ",")
+
+	for _, env := range EnvironmentKeyValues {
+		splitEnv := strings.Split(env, "=")
```

Comment:
can be dangerous ? We can't insert = inside our secret or key?

Reply:
I couldn't find a way to have something else than strings for GitHub Action inputs :(

Reply:
can you add some comment inside readme.md and action.yaml to inform that we can't use '=' and ',' inside key/value actually